app/views/providers/capital_assessment_results/show.html.erb

```diff
@@ -8,9 +8,7 @@
     <%= render @result_partial %>
   </div>
 
-  <%= render 'result_details' %>
-
-  <br><br>
```

Comment:
Someone obviously wanted a space after the result_details partial, is this being added anywhere?

Reply:
Not arguing that a double line break is 'good' spacing, but it may be worth checking with design if they are happy for it to be removed, or if it should be replaced with a `govuk-!-padding-bottom-6` on the accordion itself?
![image](https://user-images.githubusercontent.com/6757677/196675027-250804c4-5d56-4277-8809-736e011805e2.png)